src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Properties'                          : 'Propriétés',
+            'Numbers'                             : 'Nombres',
+            'Special Characters'                  : 'Caractères spéciaux',
+            'More Options'                        : 'Plus d’options',
+            'Favorite'                            : 'Favori',
+            'Encryption'                          : 'Chiffrement',
+            'On the server'                       : 'Sur le serveur',
+            'Custom Fields'                       : 'Champs personnalisés',
+            'Secret'                              : 'Secret',
+            'File'                                : 'Fichier',
+            'Value'                               : 'Valeur',
+            'Toggle visibility'                   : 'Changer la visibilité',
+            'Generate password'                   : 'Générer un mot de passe',
+            'Simple Server Side Encryption V1'    : 'Version 1 simple chiffrement coté serveur',
+            'Take some notes'                     : 'Prendre quelques notes',
+            'Save'                                : 'Sauvegarder',
+            'Create folder'                       : 'Créer un dossier',
+            'Folder created'                      : 'Dossier crée',
+            'Creating folder failed'              : 'Le dossier n’a pas pu être créé',
+            'Rename folder'                       : 'Renommer le dossier',
+            'Folder renamed'                      : 'Dossier renommé',
+            'Renaming folder failed'              : 'Le dossier n’a pas pu être renommé',
+            'Folder moved'                        : 'Dossier déplacé',
+            'Moving folder failed'                : 'Le dossier n’a pas pu être déplacé',
+            'Delete folder'                       : 'Effacer le dossier',
+            'Do you want to delete the folder'    : 'Êtes vous sur de vouloir effacer le dossier ?',
+            'Folder deleted'                      : 'Dossier effacé',
+            'Deleting folder failed'              : 'Le dossier n’a pas pu être effacé',
+            'Folder restored'                     : 'Dossier restauré',
+            'Restoring folder failed'             : 'Le dossier n’a pas pu être restauré',
+            'Create tag'                          : 'Créer une étiquette',
+            'Tag created'                         : 'Étiquette crée',
+            'Creating tag failed'                 : 'L’étiquette n’a pas pu être effacé',
+            'Edit tag'                            : 'Modifier l’étiquette',
+            'Tag saved'                           : 'Étiquette sauvée',
+            'Saving tag failed'                   : 'L’étiquette n’a pas pu être sauvé',
+            'Delete tag'                          : 'Effacer l’étiquette',
+            'Do you want to delete the tag'       : 'Êtes vous sur de vouloir effacer l’étiquette ?',
```

Comment:
`sur` should be `sûr` in this case => `Êtes vous sûr de vouloir effacer l’étiquette ?`